code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -262,7 +262,8 @@
 
 	var/obj/item/projectile/A = new projectiletype(src.loc)
 	playsound(user, projectilesound, 100, 1)
-	if(!A)	return
+	if(!A)
+		return
 
 	A.current = target
 	A.firer = src

```

Comment:
The two lines below this use colons for `x` and `y` because whoever wrote this proc was too lazy to use `atom/` instead of `var/`, despite the fact that this proc only gets called with atom arguments for `target` and `start`. That said, I'd rather not touch even more code with this PR.